README.md

```diff
@@ -182,8 +183,10 @@ let () =
 Use [Random module](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Random.html) to generate random numbers
 
 ```ml
-let () =
-  Js.log (Random.int 5)
+(* To make sure you have a different seed when your program runs each time *)
+let () = Js.Date.now () |> int_of_float |> Random.init
```

Comment:
I think putting `Random.init` first here would focus more on the purpose of this invocation:

```ml
let () = Random.init (Js.Date.now () |> int_of_float)
```